### PR TITLE
[MIG] partner_contact_company

### PR DESCRIPTION
**Pull request by JayVora-SerpentCS**

### CODE REVIEW

File: partner_contact_company/models/partner.py [hunk 1]
```diff
@@ -7,41 +7,47 @@
 
 
 class ResPartner(models.Model):
+    """Enhance the feature of Partner."""
```
Comment (sebastienbeau) [inline partner_contact_company/models/partner.py]: Can you remove this doc, I think that inheriting existing model do not need any extra documentation. It already explicite, and adding such extra documentation mean also maintaining it.

File: partner_contact_company/models/partner.py [hunk 2]
```diff
@@ -7,41 +7,47 @@
...
     def onchange_company(self):
+        """The method used to set company name to it's child."""
         if self.is_company:
             self.company = None
         elif self.parent_id.is_company:
             self.company = self.parent_id.name
 
     @api.multi
     def name_get(self):
+        """Override the method set name of partner."""
```
Comment (sebastienbeau) [inline partner_contact_company/models/partner.py]: same here, adding doc on native method, is "in my opinion" useless

File: partner_contact_company/models/partner.py [hunk 3]
```diff
@@ -56,7 +62,7 @@ def name_get(self):
                 if ctx.get('show_address'):
                     name = name + "\n" + self._display_address(
                         record, without_company=True)
-                name = name.replace('\n\n', '\n')
-                name = name.replace('\n\n', '\n')
+                if name:
+                    name = name.replace('\n\n', '\n')
```
Comment (sebastienbeau) [inline partner_contact_company/models/partner.py]: In fact we need to review that code, the purpose of this "ugly" duplicated line was to support the case where you have 4 useless "\n" and to only have "one" at the end
maybe we should do a loop
```
while "\n\n" in name:
   name = name.replace('\n\n', '\n')
```

File: partner_contact_company/__manifest__.py [hunk 4]
```diff
@@ -4,27 +4,19 @@
...
-    },
-    "depends": [
-        "base",
-    ],
-    "data": [
-        "views/partner_view.xml",
+    'name': 'Partner Company',
+    'version': '10.0.1.0.0',
+    'author': "Akretion, "
+              "Serpent Consulting Services Pvt. Ltd.",
```
Comment (sebastienbeau) [inline partner_contact_company/__manifest__.py]: I should add this in the Readme of the project (to have some guide line and avoid frustration), but I think that migrating a module is not enough to be set as an author of the module. Some special case can deserve this promotion (like huge refactoring). But for a simple migration I think is not enough. Thanks for understanding my point of view.

**Review by sebastienbeau — COMMENTED** [review]
Thanks for contributing to that project, I have done some comment.
Thanks

**Comment by bansari-serpentcs** [comment]
Thank you @sebastienbeau  to review. 
I have improved code please check it.

**Pull request merged**